openquakeplatform_building_class/templates/building-class/user-settings.html

```diff
@@ -71,9 +70,3 @@ <h2>{% trans "User settings" %}</h2>
   </div>
   <div id="dialog-message" title="Error" style="display: none; width: 600px;"></div>
 {% endblock body %}
-
-{% block extra_script %}
-    {{block.super}}
-    {% include 'search_scripts.html' %}
```

Comment:
remove just the include

Reply:
Fixed with this 828d81e0221f19983e8848ea39ebd72625749c56